tests/compile-test.rs

```diff
@@ -302,8 +291,7 @@ fn run_ui_cargo() {
 
     ui_test::run_tests_generic(
         vec![config],
-        args,
-        |path, args, _config| path.ends_with("Cargo.toml") && ui_test::default_filter_by_arg(path, args),
+        |path, config| path.ends_with("Cargo.toml") && ui_test::default_file_filter(path, config),
```

Comment:
This ought to be `default_any_file_filter` (or whatever I called it), the one that doesn't require `.rs`

Reply:
whoops fixed